### PR TITLE
Sync debugging

### PR DESCRIPTION
**Pull request by czue**
@dannyroberts @benrudolph more sync debug fun

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py [hunk 1]
```diff
@@ -51,3 +60,13 @@ def handle(self, *args, **options):
                             log_names[j],
                             ', '.join(case_diff)
                         )
+
+        if options['debugger']:
+            union_of_ids = set().union(*[set(log.get_footprint_of_cases_on_phone()) for log in logs])
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py]: F841 local variable 'union_of_ids' is assigned to but never used

File: corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py [hunk 2]
```diff
@@ -51,3 +60,13 @@ def handle(self, *args, **options):
...
+            intersection_of_ids = set().intersection(*[set(log.get_footprint_of_cases_on_phone()) for log in logs])
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py]: F841 local variable 'intersection_of_ids' is assigned to but never used

File: corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py (hunk 1 above)
Comment (czue) [inline corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py]: (these are just useful to have in local context when you drop into a debugger on the next lines)

File: corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py [hunk 3]
```diff
@@ -51,3 +60,13 @@ def handle(self, *args, **options):
...
+            import pdb
```
Comment (benrudolph) [inline corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py]: `ipdb` is a lifechanger if you aren't already using it: https://pypi.python.org/pypi/ipdb

**Comment by benrudolph** [comment]
:+1: once tests pass

**Pull request merged**